example/src/pages/ManyViewModels.tsx

```diff
@@ -91,13 +97,60 @@ export default function ManyViewModels() {
     }
   }, [riveFile]);
 
+  const handleLoadImage = async () => {
+    if (!riveViewRef.current) return;
+
+    setIsLoadingImage(true);
+    setImageError(null);
+    try {
+      const viewModelInstance = riveViewRef.current.getViewModelInstance();
+      if (!viewModelInstance) {
+        setImageError('No view model instance found in view');
+        return;
+      }
+
+      const imgProp = viewModelInstance.imageProperty('imageValue');
+      if (!imgProp) {
+        setImageError('Image property "imageValue" not found');
+        return;
+      }
+
+      const riveImage = await RiveImages.loadFromURLAsync(
+        'https://picsum.photos/id/372/500/500'
+      );
+      imgProp.set(riveImage);
+      riveViewRef.current.play();
+
+      imgProp.addListener(() => {
```

Comment:
Might be bad practice showing adding a listener and not removing it. While playing around with the example I noticed that we'd end up getting multiple calls if you spam the button.